edu.harvard.i2b2.ontology/src/core/edu/harvard/i2b2/ontology/ejb/NodeType.java

```diff
@@ -61,7 +62,17 @@ public void setType(String type) {
 	
 	public NodeType() {}
 	
-	
+    public NodeType(GetAllChildrenType allChildrenType) {
+        node = StringUtil.getTableCd(allChildrenType.getParent());
+        if (allChildrenType.getType().equals("default")) {
+            type = "core";
+        } else {
+            type = allChildrenType.getType();
+        }
+        blob = allChildrenType.isBlob();
+        parent = "mhildren";
```

Comment:
is this "mhildren" mispelling?

Reply:
I am not sure.  I copy that code directly from the existing code.  If it's a misspelling, it's also in the original code.

```java
public NodeType(GetChildrenType childrenType) {
	node = StringUtil.getTableCd(childrenType.getParent());
	if (childrenType.getType().equals("default")){
		type = "core";
	}else {
		type = childrenType.getType();
	}
	blob = childrenType.isBlob();
	parent= "mhildren";
}
```
https://github.com/i2b2/i2b2-core-server/blob/master/edu.harvard.i2b2.ontology/src/core/edu/harvard/i2b2/ontology/ejb/NodeType.java

Reply:
Follow up:  Could be ***modifierchildren***.  Mike will look into it.